CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.0)
```

Comment:
We support a few consumers with build environments that have older versions of CMake. (We have CI to ensure that [our build succeeds with CMake 3.2](https://github.com/aws/aws-lc/blob/main/.github/workflows/cmake.yml).)

Can this change be made in a way that would be supported by older versions of CMake?